components/eam/src/chemistry/mozart/mo_drydep.F90

```diff
@@ -30,119 +31,40 @@ module mo_drydep
-             xooh_ndx, ch3cho_ndx, isopooh_ndx
-  integer :: alkooh_ndx, mekooh_ndx, tolooh_ndx, terpooh_ndx, ch3cooh_ndx
-  integer :: soa_ndx, so4_ndx, cb1_ndx, cb2_ndx, oc1_ndx, oc2_ndx, nh3_ndx, nh4no3_ndx, &
-             sa1_ndx, sa2_ndx, sa3_ndx, sa4_ndx, nh4_ndx
-  integer :: soam_ndx, soai_ndx, soat_ndx, soab_ndx, soax_ndx, &
-             sogm_ndx, sogi_ndx, sogt_ndx, sogb_ndx, sogx_ndx
-
-  logical :: alkooh_dd, mekooh_dd, tolooh_dd, terpooh_dd, ch3cooh_dd
-  logical :: soa_dd, so4_dd, cb1_dd, cb2_dd, oc1_dd, oc2_dd, nh3_dd, nh4no3_dd, &
-             sa1_dd, sa2_dd, sa3_dd, sa4_dd, nh4_dd
-  logical :: soam_dd, soai_dd, soat_dd, soab_dd, soax_dd, &
-             sogm_dd, sogi_dd, sogt_dd, sogb_dd, sogx_dd
-
-  logical :: pan_dd, mpan_dd, no2_dd, hno3_dd, o3_dd, isopooh_dd, ch4_dd,&
-             h2o2_dd, onit_dd, onitr_dd, ch2o_dd, macrooh_dd, xooh_dd, &
-             ch3ooh_dd, pooh_dd, ch3coooh_dd, c2h5ooh_dd, eooh_dd, ch3cho_dd, c2h5oh_dd, &
-             c3h7ooh_dd, rooh_dd, ch3cocho_dd, co_dd, ch3coch3_dd, &
-             glyald_dd, hyac_dd, ch3oh_dd, hydrald_dd, h2_dd, Pb_dd, o3s_dd, o3inert_dd
 
   integer :: so2_ndx
-  integer :: ch3cn_ndx, hcn_ndx, hcooh_ndx
-  logical :: ch3cn_dd,  hcn_dd, hcooh_dd
-
-  integer :: o3a_ndx,xpan_ndx,xmpan_ndx,xno2_ndx,xhno3_ndx,xonit_ndx,xonitr_ndx,xno_ndx,xho2no2_ndx,xnh4no3_ndx
-  logical :: o3a_dd, xpan_dd, xmpan_dd, xno2_dd, xhno3_dd, xonit_dd, xonitr_dd, xno_dd, xho2no2_dd, xnh4no3_dd
-
-  integer :: cohc_ndx=-1, come_ndx=-1, co01_ndx=-1, co02_ndx=-1, co03_ndx=-1, co04_ndx=-1, co05_ndx=-1
-  integer :: co06_ndx=-1, co07_ndx=-1, co08_ndx=-1, co09_ndx=-1, co10_ndx=-1
-  integer :: co11_ndx=-1, co12_ndx=-1, co13_ndx=-1, co14_ndx=-1, co15_ndx=-1
-  integer :: co16_ndx=-1, co17_ndx=-1, co18_ndx=-1, co19_ndx=-1, co20_ndx=-1
-  integer :: co21_ndx=-1, co22_ndx=-1, co23_ndx=-1, co24_ndx=-1, co25_ndx=-1
-  integer :: co26_ndx=-1, co27_ndx=-1, co28_ndx=-1, co29_ndx=-1, co30_ndx=-1
-  integer :: co31_ndx=-1, co32_ndx=-1, co33_ndx=-1, co34_ndx=-1, co35_ndx=-1
-  integer :: co36_ndx=-1, co37_ndx=-1, co38_ndx=-1, co39_ndx=-1, co40_ndx=-1
-  integer :: co41_ndx=-1, co42_ndx=-1
-
-
-  integer :: &
-       o3_tab_ndx = -1, &
-       h2o2_tab_ndx = -1, &
-       ch3ooh_tab_ndx = -1, &
-       co_tab_ndx = -1, &
-       ch3cho_tab_ndx = -1
-  logical :: &
-       o3_in_tab = .false., &
-       h2o2_in_tab = .false., &
-       ch3ooh_in_tab = .false., &
-       co_in_tab = .false., &
-       ch3cho_in_tab = .false.
-
-  real(r8), parameter    :: small_value = 1.e-36_r8
+
+  ! Note for C++ port: constant rair and grav are defined locally.
+  ! Use shared ones from physconst will result in NBFB
   real(r8), parameter    :: large_value = 1.e36_r8
   real(r8), parameter    :: diffm       = 1.789e-5_r8
   real(r8), parameter    :: diffk       = 1.461e-5_r8
   real(r8), parameter    :: difft       = 2.060e-5_r8
-  real(r8), parameter    :: vonkar      = karman
   real(r8), parameter    :: ric         = 0.2_r8
```

Comment:
Parameters that are used only in one subroutine, should be moved in that subroutine so that we know all the input in one place. Could you please move `diffm`, `diffk`, `difft`, and `ric` parameters where they are used?

Reply:
`diffm` and `difft` moved to `dvel_inti_fromlnd`
`ric` moved to `drydep_xactive`